test/parallel/test-string-decoder.js

```diff
@@ -124,13 +124,15 @@ assert.strictEqual(decoder.write(Buffer.from('3DD8', 'hex')), '');
 assert.strictEqual(decoder.write(Buffer.from('4D', 'hex')), '');
 assert.strictEqual(decoder.end(), '\ud83d');
 
+const expectedError = common.expectsError('ERR_UNKNOWN_ENCODING', Error);
```

Comment:
These arguments are not correct for `expectsError`. Please check that in the function definition. You can also compare the handling in other test files.